code/game/machinery/machinery.dm

```diff
@@ -310,6 +310,12 @@ Class Procs:
 
 	return ..()
 
+/obj/machinery/tgui_status(mob/user, datum/tgui_state/state)
```

Comment:
Machinery had a `CanUseTopic` override for this, but thats part of nano, which will be axed, so I am doing the cleanup now